src/CountryHeader.sass

```diff
@@ -0,0 +1,8 @@
+@import './App.sass'
+
+#country-container
+  margin: 5px auto auto auto
+
+h1
```

Comment:
Again, might run on every h1, indent it to run inside the context of `#country-container`